docs/src/examples/components/Segment/Variations/SegmentExampleInverted.shorthand.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react'
+import { Segment, Text } from '@stardust-ui/react'
+
+const SegmentExampleInvertedShorthand = () => (
+  <div>
+    <Segment content="Colored segment." variables={siteVars => ({ color: siteVars.brand })} />
+    <br />
+    <Segment
+      inverted
+      content={<Text content="Colored inverted segment." styles={{ color: 'white' }} />}
```

Comment:
what if brand color would be `white` here as well? would better suggest just to hardcode colors for now

Reply:
I do not totally agree with hardcoding; we should use `siteVars` in order to leverage switching themes capabilities; I think we should create an issue on using only predefined colors in examples after we have a default theme; what do you think? @kuzhelov @levithomason 
for now I'll hardcode colors

Reply:
agree on both comments here.  we should hard code for now only because we lack a sufficient site variables interface.  once we have something for background and foreground in the site variables interface, we should consistently use only site variables.

Reply:
done